### ISSUE TITLE
[Feature Request] Wormhole signature delete timer

### ISSUE THREAD

**Issue by iXav** [exodus4d/pathfinder#833]
Hi,

All wormholes have a maximum life time, it would be great if when a wormhole type sig is pasted into a map, it would automaticaly create a deletion timer (default 24hrs). Also would be great to be able to customize this timer.

Cheers

**Comment by porowns** [comment]
No, it wouldn't. Wormholes can remain unspawned if you warp to them, the only mechanism that should delete a signature is lazy delete.
